SpatialGDK/Source/SpatialGDK/Private/LoadBalancing/LayeredLBStrategy.cpp

```diff
@@ -46,15 +46,9 @@ void ULayeredLBStrategy::Init()
 		UE_LOG(LogLayeredLBStrategy, Log, TEXT("Creating LBStrategy for Layer %s."), *LayerName.ToString());
 		for (const TSoftClassPtr<AActor>& ClassPtr : LayerInfo.ActorClasses)
 		{
-			if (ClassPtr.IsValid())
-			{
-				UE_LOG(LogLayeredLBStrategy, Log, TEXT(" - Adding class %s."), *ClassPtr->GetName());
-				ClassPathToLayer.Add(ClassPtr, LayerName);
-			}
-			else
-			{
-				UE_LOG(LogLayeredLBStrategy, Log, TEXT(" - Invalid class not added %s"), *ClassPtr.GetAssetName());
-			}
+			UE_LOG(LogLayeredLBStrategy, Log, TEXT(" - Adding class %s."), *ClassPtr.GetAssetName());
```

Comment:
Maybe would be nice to log the layer name here as well.

Reply:
The layer name is logged above, so the output ends up looking like:

Creating LBStrategy for Layer default:
 - Adding class foo
 - Adding class bar

Reply:
Ah, nice.